src/types/BeachballOptions.ts

```diff
@@ -83,6 +83,10 @@ export interface RepoOptions {
      */
     postpublish?: (packagePath: string, name: string, version: string) => void | Promise<void>;
   };
+
+  transform?: {
```

Comment:
I think some documentation would be good here like `hooks` above

Reply:
Yes, sure. I have added a description like we have for the `hooks`.